src/Utils.hh

```diff
@@ -21,8 +21,10 @@
 #include <string>
 #include <optional>
 #include <vector>
+#include "sdf/Console.hh"
```

Comment:
nit: Is this needed?

Reply:
I thought it was for sdfwarn/sdfdbg, but I guess not. Removed